elmslie-core/src/main/java/vivid/money/elmslie/core/store/ElmStore.kt

```diff
@@ -62,8 +65,12 @@ class ElmStore<Event : Any, State : Any, Effect : Any, Command : Any>(
         scope.launch {
             try {
                 logger.debug("New event: $event")
```

Comment:
Can we move this line into Mutex scope? Will be more fair))

Reply:
I thought about it but decided that more important is when the event arrived at this method. Are you sure that it will be better inside the scope? If so I'll move it.

Reply:
For me, this log indicates that event is received by reducer and handling it. Well I don't know which side is better)) Let's keep it as is)